### PR TITLE
register nyanbrowser.is-a.dev

### PR DESCRIPTION
**Pull request by hoppygamer**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [ ] The file is in the `domains` folder and is in the JSON format.
- [ ] The file's name is all lowercased and alphanumeric. 
- [ ] You have completed your website. 
- [ ] The website is reachable.  
- [ ] The CNAME record doesn't contain `https://` or `/`.  
- [ ] There is sufficient information at the `owner` field.  

## Website Link/Preview

image

### CODE REVIEW

File: domains/nyanbrowser.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "description": "nyanbrowser.is-a.dev",
```
Comment (user0-07161) [inline domains/nyanbrowser.json:2]:
```suggestion
  "description": "Website to download nyanbrowser",
```

File: domains/nyanbrowser.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  "repo": "https://github.com/hoppygamer/nyanbrowser.github.io",
```
Comment (user0-07161) [inline domains/nyanbrowser.json:3]:
```suggestion
  "repo": "https://github.com/nyanbrowser/nyanbrowser.github.io",
```
Moved to new repo

**Comment by user0-07161** [comment]
@DEV-DIBSTER what

**Comment by DEV-DIBSTER** [comment]
My bad. @user0-07161

**Comment by user0-07161** [comment]
oh alr

**Pull request merged**